css/signin.css

```diff
@@ -171,18 +276,92 @@ body {
     display: block;
 }
 
+/* Container class for footer */
+.container {
+    max-width: 1200px;
```

Comment:
happy to keep this as 1200px as it would mean the footer for larger screens would create nice margin so happy to keep this!